helper_extension/inject.js

```diff
@@ -0,0 +1,34 @@
+// this is the code which will be injected into a given page...
+// Chrom extension code skeleton from https://gist.github.com/danharper/8364399
```

Comment:
Typos/grammar

Reply:
This is from the Chrome extension template I borrowed... but I'll fix it